docs/techincal-guide.md

```diff
@@ -0,0 +1,397 @@
+	initialized();
+	executionDelay = _executionDelay;
+}
+```
+
+<br />
+
+## Forwarding
+
+### isForwarder
+
+This makes the Delay app an [Aragon forwarder](https://hack.aragon.org/docs/forwarding-intro).
+```
+function isForwarder() external pure returns (bool) {
+	return true;
+}
+```
+
+### canForward
+
+This grants an address the ability to create create delayed forwarding intents via the Delay app.
+```
+function canForward(address _sender, bytes) public view returns (bool) {
+	return canPerform(_sender, DELAY_EXECUTION_ROLE, arr());
+}
+```
+
+### forward
+
+This allows an external account with the `DELAY_EXECUTION_ROLE` to create a delayed intent to forward.
+```
+/**
+* @notice Store script `_evmCallScript` for delayed execution
+* @param _evmCallScript The script that can be executed after a delay
+*/
+function forward(bytes _evmCallScript) public {
+	require(canForward(msg.sender, _evmCallScript), ERROR_CAN_NOT_FORWARD);
+	_delayExecution(_evmCallScript);
+}
+```
+
+<br />
+
+## Executing Delayed Scripts
+
+### setExecutionDelay
+
+This sets the global `_executionDelay` variable that was set at initialization.
+
+Only addresses that have been given the `SET_DELAY_ROLE` are allowed to call this function.
+```
+/**
+* @notice Set the execution delay to `_executionDelay`
+* @param _executionDelay The new execution delay
+*/
+function setExecutionDelay(uint256 _executionDelay) external auth(SET_DELAY_ROLE) {
+	executionDelay = _executionDelay;
+}
+```
+
+### canExecute
+
+This returns a boolean that tells the caller if a certain `scriptId` can or cannot be executed.
+```
+/**
+* @notice Tells whether a script with ID #`_scriptId` can be executed or not
+* @param _scriptId The ID of the script to execute
+*/
+function canExecute(uint256 _scriptId) public scriptExists(_scriptId) returns (bool) {
+
+	if (_isExecutionPaused(_scriptId))
+			return false;
+
+	bool withinExecutionWindow = now > delayedScripts[_scriptId].executionTime;
+	return withinExecutionWindow;
+}
+```
+
+### execute
+
+This checks if the `_delayedScriptId` can be executed (has the delay window passed). If so, then it runs the script and deletes it from the `delayedScripts` mapping.
+
+An event is emitted upon successful execution.
+```
+/**
+* @notice Execute the script with ID `_delayedScriptId`
+* @param _delayedScriptId The ID of the script to execute
+*/
+function execute(uint256 _delayedScriptId) external {
+	require(canExecute(_delayedScriptId), ERROR_CAN_NOT_EXECUTE);
+
+	runScript(delayedScripts[_delayedScriptId].evmCallScript, new bytes(0), new address[](0));
+
+	delete delayedScripts[_delayedScriptId];
+
+	emit ExecutedScript(_delayedScriptId);
+}
+```
+
+<br />
+
+## Delaying Execution
+
+### delayExecution
+
+This is a wrapper for the `_delayExecution` function. This allows external accounts to call that function, but only if they have the `DELAY_EXECUTOIN_ROLE`. This is intended to be used so that other Aragon apps can created delayed forwarding intents.
```

Comment:
Not sure if I fully understand, but I updated 228 to reflect that forward is called by EOAs. Is it accurate now, or does it still need more work?

Reply:
I would remove the las part `This is intended to be used so that other Aragon apps can created delayed forwarding intents.` since the function that will most likely be called form other Aragon apps is `forward`